helm/fasttrackml/values.yaml

```diff
@@ -0,0 +1,101 @@
+---
+replicaCount: 1
+
+image:
+  repository: gresearch/fasttrackml
+  pullPolicy: IfNotPresent
+  # Overrides the image tag whose default is the chart appVersion.
+  tag: "latest"
+
+imagePullSecrets: []
+nameOverride: ""
+fullnameOverride: ""
+
+data:
+  # common service settings.
+  FML_DEV_MODE: true
+  FML_LOG_LEVEL: DEBUG
+
+  # database settings.
+  LC_COLLATE: POSIX
+  POSTGRES_USER: postgres
```

Comment:
POSTGRES settings are not really part of this service. We put user/pwd/host/db in the FML_DATABASE_URI for postgres.